src/constants/Countries.ts

```diff
@@ -1,6 +1,5 @@
-export const WORLDAVERAGE = 421;
-
 export const COUNTRIES = {
```

Comment:
unrelated but I think this needs to be named `CARBON_EMISSIONS_PER_COUNTRY` because it's a little unclear what this is if possible?